src/index.html

```diff
@@ -1,17 +1,184 @@
+            <input type="checkbox" name="terms">
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration:</legend>
+        <div class="form-field">
+          <label>
+            E-mail:
+            <input 
+              type="email" 
+              name="email" 
+              placeholder="email@example.com" 
+              required
+            >
+          </label>
+        </div>
+      
+        <div class="password">
+          <label>
+            Password:
+            <input 
+              type="password" 
+              name="password" 
+              required
+            >
+          </label>
+        </div>
+    </div>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input type="radio" name="cat">
+            Yes
+          </label>
+          <label>
+            <input type="radio" name="cat">
+            No
+          </label>
+        </div>
+      
+        <div class="form-field">
+          <label for="color">What is your favorite color?
+            <input 
+              type="color" 
+              name="color" 
+              value="#000000"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What time do you go to bed?
+            <input 
+              type="time" 
+              name="timeToBed" 
+              step="2"
+            >
+          </label>
+        </div>
+      
+        <div class="form-field">
+          <label for="carBrand">What are your favorite brands of cars?
+            <select name="carBrand" multiple>
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="lada">lada</option>
+            </select>
+          </label>
+        </div>
+      
+        <div class="work">
+          <label>
+            How do you rate our work?
+            <input 
+              type="range" 
+              name="rate" 
+              min="0" 
+              max="100"
+              value="0"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Additional info:</legend>
+        <div class="form-field">
+          <label for="comments">Comments:
+            <textarea class="comment" 
+          name="comments" 
+          minlength="50" 
+          maxlength="300"
+          >
+            </textarea>
+          </label>
+        </div>
+      
+        <div class="reccomend">
+          <label for="reccomend">Would you reccomend us?
+            <select name="reccomend">
+              <option value="yes">yes</option>
+              <option value="no">no</option>
+            </select>
+          </label>
+        </div>
+      </fieldset>
+      <div>
+        <button type="submit">Submit</button>
+      </div>
```

Comment:
Same. Don't add additional wrappers everywhere 
```suggestion
        <button type="submit">Submit</button>
```